documentation/sile.sil

```diff
@@ -1668,7 +1669,6 @@ Now we can finally complete our implementation of centering:
 \\set[parameter=document.rskip,value=0pt plus 100000pt]
 \\set[parameter=document.spaceskip,value=0.5en]
 \\set[parameter=current.parindent,value=0pt]
-\\set[parameter=document.parindent,value=0pt]
```

Comment:
Why did you drop this line? Without it the example will only work for one input paragraph instead of for several.

Reply:
Oh. This seems to have been naïve of me but I deleted it because the actual usage of this centering formulation in the following lines [doesn't have this line](https://github.com/simoncozens/sile/pull/594/files#diff-58dba0558da425ff2c2d4e38a9c14421R1681). Now I realize this is because it is not needed only because there is only one example paragraph but it would be needed elsewhere. Please feel free to edit my PR.

Reply:
Fixed. I amended your commit to resolve this and fix the commit message.